<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>641</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter Client</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>461</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>200</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>.....</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>281</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>nom:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nom"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>prenom:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="prenom"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>email:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="email"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>N Telephone:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="numeroTelephone"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>CNIE:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="CNIE"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Chercher Client</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetClients_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>601</width>
        <height>291</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID Client</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prenom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>N Telephone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CNIE</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>451</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>CNIE:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="CNIECherche"/>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonChercher">
         <property name="text">
          <string>Chercher</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Afficher Client</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetClients">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>10</y>
        <width>601</width>
        <height>291</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID Client</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prenom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>N Telephone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CNIE</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="ButtonMise">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Mise Ã  jour</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Supprimer Client</string>
     </attribute>
     <widget class="QLabel" name="message">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>411</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>CNIE :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="messagesupp"/>
       </item>
       <item>
        <widget class="QPushButton" name="supprimerButton">
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Page d'acceuille</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
